package.json

```diff
@@ -22,8 +22,9 @@
     "prepublishOnly": "npm run precompile",
     "postinstall": "node ./scripts/check_rfcs/check_rfcs.js",
     "watch": "npm run clean-lib && npm run copy-files -- --watch & npm run babel -- --watch",
+    "build-storybook-cypress": "cross-env STORYBOOK_BUILD=true NODE_OPTIONS=--openssl-legacy-provider build-storybook -c .storybook",
```

Comment:
You can add this to the normal `build` command.

Reply:
It's not possible to add this to the build-storybook script as the action used in the chromatic workflow is using node 16